src/web/screens/AffiliationViews/DeploymentView/DetailsView/SkapJobView/SkapJobTable.tsx

```diff
@@ -17,8 +17,8 @@ interface IBigipJobTable extends IBigipJob {
 }
 
 interface ISkapJobTableProps {
-  websealJobs: IWebsealJobTable[];
-  bigipJobs: IBigipJobTable[];
+  websealJobs: any[];
```

Comment:
Litt dumt å ikke beholde typene her. Evt burde vel interfacene fjernes hvis det ikke går an å gjøre noe med. 
Eller er det plan for å akseptere typer i designsystemet i en fremtidig versjon?

Reply:
Enig, klarer hvertfall ikke å finne en løsning slik designsystemet er nå med å bruke de typene.
Om det skal aksepteres typer i fremtidige versjoner, vet jeg ikke 100%. Jeg har jo ikke sagt ennå at det er noe vi må ha.

Reply:
Hvis **data** parameteret til Table krever any så ville jeg heller endret typen der fremfor å sette den til any her.
Eks: `data={websealJobs as any}` da bevarer vi typene så lenge som mulig.

Reply:
helt enig med forslaget, jeg skal fikse